static/src/javascripts/projects/common/modules/commercial/hosted-video.js

```diff
@@ -19,8 +19,7 @@ define([
     videojs,
     loadingTmpl
 ) {
-
-    var $videoEl = $('.vjs-hosted__video');
+    var $videoEl;
```

Comment:
The `hosted-video.js` is loaded with the whole commercial module. But because the thrasher is loaded from the DFP,  during the page load there is no `.vjs-hosted__video` yet in the DOM. So I needed to move it to the `init()` method so it will be run in Promise, after being attached to the body.